index.html

```diff
@@ -171,5 +171,31 @@ <h3>Skills</h3>
                 </ul>
     </div>
 </section>
+<section class="form-flex">
+<p class="text-contact">I'm always interested 
+    in hearing about new projects, so if you'd 
+    like to chat please get in touch.
+</p>
```

Comment:
- [ ] While you had used great effort to implement the form paragraph. I suggest that you stick to the Figma design. By implementing the following, the paragraph will look professional.
   - Ensure that the color of the text looks similar to the one of the Figma design. 
   - Ensure that the text is centered.

- Your design 😎 
![p center my](https://user-images.githubusercontent.com/99394374/214513068-858ee54e-de48-469e-bd56-fa82dd061626.PNG)

- Figma design 😎 
![p center](https://user-images.githubusercontent.com/99394374/214513128-682e927b-cced-4876-ab7b-c3fb17aca8e9.PNG)